pages/queries.rst

```diff
@@ -45,6 +45,10 @@ Messages that do not have the field *type*::
 
   NOT _exists_:type
 
+Messages that match RegExp *FastEthernet[0-9]+* (ElasticSearch will convert messages to lowercase)::
```

Comment:
Please use back-quotes to emphasize the regular expression:
http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

```text
``FastEthernet[0-9]+``
```